testing/tests/filters.rs

```diff
@@ -38,6 +38,18 @@ fn filter_format() {
     assert_eq!(t.render().unwrap(), "\"formatted\"");
 }
 
+#[derive(Template)]
+#[template(path = "fmt.html", escape = "none")]
```

Comment:
Please inline the source here (so `source = "{{ var|fmt(\"{:?}\") }}"` rather than `path = `).